### ISSUE TITLE
Support --include_imports equivalent in proto_library

### ISSUE THREAD

**Issue by djudd-stripe** [bazelbuild/rules_proto#57]
### Description of the feature request:

We would find it useful to have an attribute on the native `proto_library` rule which causes it to behave like protoc when passed `--include_imports`, which would include all the transitive descriptor set contents in the output descriptor set. Supporting self-contained descriptor set outputs in this way would make it easier to write tooling that uses descriptor sets as the import/export unit for passing schemas between repos, which in our setting are not always built with Bazel, and it would bring the rule closer to feature parity with using protoc directly.

**Comment by Yannic** [comment]
Duplicating into #45 

TL;DR `rules_proto` should provide a rule for this (similar to @drigz's rule above).
